src/components/home/LandingTitle.jsx

```diff
@@ -1,5 +1,25 @@
+import * as React from "react";
+import Icon from "public/HLG-logo.png";
+import Button from "src/components/Button";
+import Image from "next/image";
+
+console.log(Icon);
+
 const LandingTitle = () => {
-  return <div>LandingTitle</div>;
+  return (
+    <div className="h-screen bg-gradient-to-r flex items-center">
+      <Image src={Icon} alt="Icon" className="w-1/12" />;
+      <div className="flex-call">
```

Comment:
flex-col (as in rows and cols)

 and remove the margins you added inside this div's children